interproscan/5.72-103.0/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM mambaorg/micromamba:2.0.5
+
+ARG VERSION=5.72-103.0
+
+LABEL software="InterProScan"
+LABEL software.version="5.72-103.0"
+LABEL description="InterProScan is a search engine for InterPro is a database which integrates together predictive information about proteins' function."
+LABEL website="https://github.com/ebi-pf-team/interproscan"
+LABEL license="Apache 2.0"
+
+COPY --chown=$MAMBA_USER:$MAMBA_USER env.yml /tmp/env.yml
+
+RUN micromamba install -y -n base -f /tmp/env.yml \
+    && micromamba install -y -n base conda-forge::procps-ng \
+    && micromamba env export --name base --explicit > environment.lock \
```

Comment:
Just exporting the .lock file for provenance purposes?

Reply:
Yeah, I didn't pin the versions of he tools in the dockerfile (which I took from the conda recepie). So I just exported the lock file